### PR TITLE
MSF-25372: Upgrade lcm-bricks image to C9S

### PR DESCRIPTION
**Pull request by vietnguyenkim-gooddata**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1555/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1404/ : FAILURE in 54s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1556/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1405/ : SUCCESS in 9m 51s

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/40/ : SUCCESS in 16m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1557/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/35/ : SUCCESS in 16m 19s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/38/ : FAILURE in 9m 50s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/35/ : FAILURE in 1h 05m 44s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/40/ : SUCCESS in 21m 51s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/38/ : SUCCESS in 14m 42s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1579/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1421/ : FAILURE in 42s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1580/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1422/ : FAILURE in 7m 48s

**Comment by vietnguyenkim-gooddata** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1584/ : SUCCESS in 5s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1425/ : SUCCESS in 8m 32s

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/49/ : SUCCESS in 13m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1585/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/44/ : SUCCESS in 16m 09s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/47/ : FAILURE in 11m 08s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/45/ : FAILURE in 41m 33s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/49/ : SUCCESS in 20m 13s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/47/ : SUCCESS in 13m 29s

**Comment by hung-nguyen-hoang** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/54/ : FAILURE in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1595/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/49/ : SUCCESS in 15m 12s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/52/ : FAILURE in 8m 41s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/50/ : FAILURE in 1h 06m 09s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/54/ : FAILURE in 1m 18s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/52/ : FAILURE in 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1598/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1431/ : SUCCESS in 8m 56s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1599/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1432/ : SUCCESS in 9m 01s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1600/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1433/ : FAILURE in 58s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1601/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1434/ : FAILURE in 1m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1602/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1435/ : FAILURE in 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1603/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1436/ : FAILURE in 56s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1604/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1437/ : FAILURE in 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1605/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1438/ : FAILURE in 59s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1606/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1439/ : FAILURE in 58s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1607/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1440/ : FAILURE in 1m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1608/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1441/ : FAILURE in 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1609/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1442/ : SUCCESS in 8m 49s

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1610/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1443/ : SUCCESS in 8m 48s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/57/ : FAILURE in 1m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1611/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/52/ : SUCCESS in 15m 48s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/55/ : FAILURE in 10m 33s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/53/ : FAILURE in 1h 07m 00s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/57/ : FAILURE in 33s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/56/ : FAILURE in 30s

**Comment by ngocmanhha** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/59/ : FAILURE in 1m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1612/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/53/ : SUCCESS in 16m 56s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/56/ : FAILURE in 11m 48s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/54/ : FAILURE in 1h 07m 07s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/58/ : SUCCESS in 22m 39s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/57/ : FAILURE in 49s

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/61/ : FAILURE in 2m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1614/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/55/ : FAILURE in 33s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/58/ : FAILURE in 2m 24s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/56/ : FAILURE in 2m 34s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/60/ : FAILURE in 1m 51s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/59/ : FAILURE in 1m 45s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1616/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1445/ : FAILURE in 1m 25s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1618/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1447/ : SUCCESS in 8m 50s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1619/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1448/ : SUCCESS in 9m 26s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1620/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1449/ : SUCCESS in 9m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1621/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/1450/ : SUCCESS in 8m 57s

**Comment by vietnguyenkim-gooddata** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/62/ : SUCCESS in 14m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1622/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/56/ : SUCCESS in 16m 39s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/59/ : FAILURE in 10m 40s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/57/ : FAILURE in 1h 10m 24s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/61/ : SUCCESS in 22m 20s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/60/ : SUCCESS in 13m 25s

**Comment by tqtu** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-userprov-zuul/63/ : SUCCESS in 15m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/1623/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-slow-zuul/57/ : SUCCESS in 18m 25s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-others-zuul/60/ : FAILURE in 11m 57s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-integration-tests-zuul/58/ : FAILURE in 1h 09m 13s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-release-zuul/62/ : SUCCESS in 23m 27s
- https://checklist.intgdc.com/job/lcm/job/gooddata-lcm-ruby-integration-e2e-zuul/61/ : SUCCESS in 14m 28s

**Comment by tqtu** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 ARG JRUBY_VERSION=9.4.1.0
 
 LABEL image_name="GDC LCM Bricks"
 LABEL maintainer="LCM <lcm@gooddata.com>"
 LABEL git_repository_url="https://github.com/gooddata/gooddata-ruby/"
-LABEL parent_image="020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-java-8-jre-centos8:202308081603.4a43db3"
+LABEL parent_image="020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-java-11-jre-centos9:202310031054.5119496"
 
 # which is required by RVM
-RUN yum install -y curl which patch make git maven procps \
```
Comment (tqtu) [inline Dockerfile:12]: Curl is available on this image

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ruby-gate-PR1940

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-sonarqube-scan-zuul-gate/150/ : SUCCESS in 2m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul/42/ : FAILURE in 1m 53s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/692/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-gitops-deploy-pipeline/114/ : SUCCESS in 54s

**Pull request merged**